src/test-api/api.yaml

```diff
@@ -139,19 +159,40 @@ components:
         idType:
           type: string
           enum:
-          - MSISDN
-          - EMAIL
-          - PERSONAL_ID
-          - BUSINESS
-          - DEVICE
-          - ACCOUNT_ID
-          - IBAN
-          - ALIAS
-
+            - MSISDN
+            - EMAIL
+            - PERSONAL_ID
+            - BUSINESS
+            - DEVICE
+            - ACCOUNT_ID
+            - IBAN
+            - ALIAS
         idValue:
           type: string
           minLength: 1
           maxLength: 128
+        extensionList:
+          $ref: '#/components/schemas/ExtensionList'
+    Extension:
```

Comment:
@lazolalucas  you have added Extensions and ExtensionList but i don't see it being referenced anywhere in the yaml??

Reply:
Yes thanks I forgot to remove this

Reply:
Extension is the object referenced in ExtensionList the array which in turn referenced in the party component

Reply:
Extension is the object referenced in ExtensionList the array which in turn referenced in the party component